src/js/edu-benefits/1990n/components/IntroductionPage.jsx

```diff
@@ -14,47 +14,36 @@ class IntroductionPage extends React.Component {
     return (
       <div className="schemaform-intro">
         <FormTitle title="Apply for education benefits under the National Call to Service program"/>
-        <p>This application is equivalent to Form 22-1990N (Application for VA Education Benefits Under the National Call to Service (NCS) Program).</p>
-        <SaveInProgressIntro
-          messages={this.props.route.formConfig.savedFormMessages}
-          pageList={this.props.route.pageList}
-          resumeOnly
-          {...this.props.saveInProgressActions}
-          {...this.props.saveInProgress}/>
+        <p>Equal to VA Form 22-1990N (Application for VA Education Benefits Under the National Call to Service (NCS) Program).</p>
```

Comment:
Want to make sure the removal of the SIP here was deliberate.

Reply:
Same as above, that `<SaveInProgressIntro/>` component should be added back.